### ISSUE TITLE
Bug: error: `CoinbaseWalletSDK.configure` should be called only once on iOS

### ISSUE THREAD

**Issue by felipecamposfabel** [MobileWalletProtocol/wallet-mobile-sdk#218]
### Describe the bug

After reloading the app using the React Native Developer Menu, the app freezes or crashes with the mentioned error.

### Steps

1. Run the app once on iOS
2. Use the React Native developer menu and press Reload
3. If you are using XCode, the app freezes here:
<img width="676" alt="image" src="https://user-images.githubusercontent.com/1895066/233098659-9fc71d37-188d-434d-9bc1-dd6a03b49368.png">
4. If not, the app crashes

Here's a repo, if you need, but just the basic usage should cause this.
https://github.com/felipecamposfabel/coinbase

### Expected behavior

The SDK should handle this problem, or expose a variable for us to check if the SDK is already configured.

**Comment by brianchitester** [comment]
Hey @felipecamposfabel, thanks for reporting this bug.  We've just published a fix in version 1.0.6, please update and let us know if you have any more issues.

**Comment by TheAmjadKhan** [comment]
@brianchitester  same is happening on Flutter IOS after hot restart app

```
PlatformException(configure, `CoinbaseWalletSDK.configure` should be called only once., null, null)
```
